modules/persistence/index.ts

```diff
@@ -0,0 +1,50 @@
+import * as dotenv from 'dotenv';
+// dotenv.config() should be invoked immediately, before any other imports, to ensure config is present
+dotenv.config();
```

Comment:
This can be invoked within the `connector`, but likely safest to invoke at top level, since any module/code path here might require a value from the `.env`